numpy/lib/tests/test_arraypad.py

```diff
@@ -82,6 +125,30 @@ def test_check_maximum_2(self):
             )
         assert_array_equal(a, b)
 
+    def test_check_maximum_stat_length(self):
```

Comment:
It seems to be a common misconception, but spaces around *all* binary operators is not in PEP 8.  Take a look at at https://www.python.org/dev/peps/pep-0008/#other-recommendations, and note the examples under "Yes".